internal/handlers/hana/hanadb/pool.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hanadb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "SAP/go-hdb/driver" //nolint:goimports // goimports and gci are conflicting
```

Comment:
It should imported as `github.com/SAP/go-hdb/driver`

Reply:
See other comment